Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.yml

```diff
@@ -516,7 +519,130 @@ script:
     - contextPath: Illusive.Event.status
       description: The status command ( Done, InProgress)
       type: String
-  dockerimage: demisto/python3:3.8.2.6981
+  - arguments:
+    - default: false
+      description: The incident ID on which to retrieve the associated events
+      isArray: false
+      name: incident_id
+      required: true
+      secret: false
+    - default: false
+      description: The maximum number of events to retrieve
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      description: Use offset and limit for pagination
+      isArray: false
+      name: offset
+      required: false
+      secret: false
+    deprecated: false
+    description: Retrieve all the events that are associated with an incident
```

Comment:
```suggestion
    description: "Retrieve all the events that are associated with an incident"
```

Reply:
why should I add double quotes ?

Reply:
it's just an example for all the places you use `'` in the description
i think we talked about it when your internet dropped

Reply:
you mean the change from ’s to 's ?

Reply:
no, i mean that if you do:
"some text with ' and more text" - will work

while this will probably not work:
some text with ' and more text

Reply:
it seems to be working without   ""